src/dbt/kippmiami/models/fldoe/staging/stg_fldoe__fsa.sql

```diff
@@ -88,7 +88,7 @@ with
 
             cast(regexp_extract(test_name, r'^\w+\s(\d+)') as int) - 1 as academic_year,
 
-            regexp_extract(test_name, r'^(\w+)\s') as administration_round,
+            proper(regexp_extract(test_name, r'^(\w+)\s')) as administration_round,
```

Comment:
does this work? I think the function you're looking for is `initcap`

Reply:
Whoops, yes! I forgot I was going to test that. I instead just typed 'Spring'. Should I leave the 'Spring' or use this instead. Everything else says 'Spring,' so I just need it to be in proper case, not uppper.

Reply:
if you need to pull the round from the test name and put it in proper case, use `initcap`